### PR TITLE
done

### PR DESCRIPTION
**Pull request by ammardab3an**

### CODE REVIEW

File: musicplatform/RESULTS.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+def create_albums_two_ways():
+    #first way
+    ar_ammar = Artist.objects.get(pk="ammar")
+    al0 = Album(name="ammars_first_album", artist=ar_ammar, 
+                creation_date=timezone.now()-datetime.timedelta(days=1), 
+                release_date=timezone.now()+datetime.timedelta(weeks=1), 
+                cost=20.50)
+    al0.save()
+    
+    #second way
```
Comment (yousefokasha12) [inline musicplatform/RESULTS.md:78]: This is supposed to be the second way:
``` python
drake.album_set.create(name='Take Care', release_date='2015-11-20T15:58:44+00', cost=9.99) 
```

File: musicplatform/RESULTS.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+album name: ammars_first_album, artist: ammar
+album name: ammars_second_album, artist: ammar
+```
+
+##### 5. latest release date album.
+
+code:
+
+```python
+def latest_release_album():
```
Comment (yousefokasha12) [inline musicplatform/RESULTS.md:101]:
```python
Album.objects.all().latest('release_date')
```

File: musicplatform/RESULTS.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+artist name: majed, social-media: fb.com/madjed
+	 no albums yet.
+```
+
+##### 9. list all artists with their albums (2nd way).
+
+code:
+
+```python
+def list_artists_with_albums_nd_way():
```
Comment (yousefokasha12) [inline musicplatform/RESULTS.md:216]:
```python
artists = Artist.objects.all()
for artist in artists:
    artist.album_set.all()
```

File: musicplatform/artists/models.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from unittest.util import _MAX_LENGTH
+from django.db import models
+
+class Artist(models.Model):
+    
+    name = models.CharField(max_length=200, primary_key=True)
+    social_media_link = models.CharField(max_length=300, blank=True, null=False)
```
Comment (yousefokasha12) [inline musicplatform/artists/models.py]: should be URLField

File: musicplatform/RESULTS.md (hunk 3 above)
Comment (ammardab3an) [reply musicplatform/RESULTS.md:216]: @yousefokasha12 
isn't that the same as in:
list_artists_with_albums_st_way
?

**Pull request merged**